blackbox/docs/sql/administration/ingestion_rules.txt

```diff
@@ -0,0 +1,92 @@
+the ``CREATE`` and ``DROP INGEST RULE`` statements, when the
+:ref:`administration_user_management` feature is enabled.
+
+Creating and Deleting Ingestion Rules
+.....................................
+
+Suppose we have installed an ingestion implementation for an ``mqtt`` broker,
+and we would like to filter temperature data streams by topic and store them to
+``temp_data.raw``::
+
+    cr> CREATE INGEST RULE temperature
+    ... ON mqtt
+    ... WHERE topic like 'temperature/%'
+    ... INTO temp_data.raw;
+    CREATE OK, 1 row affected (... sec)
+
+It is possible to create and drop ingestion rules even if the corresponding
+ingestion implementation is not available. However, the target table
+needs to be created manually before the ``CREATE INGEST RULE`` statement is
+issued, and it needs to have the same structure as the ingestion
+implementation's data structure.
+
+Multiple rules can be created for the same ingestion source, for example, to
+store the humidity data streams in ``humidity_data.raw``::
+
+    cr> CREATE INGEST RULE humidity
+    ... ON mqtt
+    ... WHERE topic like 'humidity/%'
+    ... INTO humidity_data.raw;
+    CREATE OK, 1 row affected (... sec)
+
+See :ref:`create-ingest-rule` for more information on ingestion rule creation.
+
+To drop the ``temperature`` ingestion rule created above::
+
+	cr> DROP INGEST RULE temperature;
+    DROP OK, 1 row affected (... sec)
+
+See :ref:`drop-ingest-rule` for more information on deleting ingestion rules.
+
+Listing Ingestion Rules
+.......................
+
+CrateDB exposes ingestion rules ``information_schema.ingestion_rules`` table.
+
+By querying the ``information_schema.ingestion_rules`` table you can get all
+information regarding the existing ingestion rules. E.g.::
+
+    cr> SELECT * FROM information_schema.ingestion_rules order by rule_name;
+    +-------------+---------------+-------------------+-------------------------+
+    | rule_name   | source_ident  | target_table      | condition               |
+    +-------------+---------------+-------------------+-------------------------+
+    | humidity    | mqtt          | humidity_data.raw | topic like 'humidity/%' |
+    +-------------+---------------+-------------------+-------------------------+
+    SELECT 1 rows in set (... sec)
+
+The column ``rule_name`` shows the identifier of the rule, the column
+``source_ident`` shows the identifier of the ingestion implementation that uses
+this rule. The column ``target_table`` identifies the target table where the
+data is stored and finally ``condition`` stands for the condition by which the
+data is filtered.
+
+Ingestion Rules and Privileges
+..............................
+
+When the :ref:`administration_user_management` is enabled, creating and
+dropping ingestion rules can only be issued by a superuser.
+
+The default user for the ``INSERT`` operations in the target ``table_ident``
+is the superuser ``crate``.
```

Comment:
I think this is an implementation detail of the ingestion source. Our interfaces currently don't enforce this